server/routes/users/friends/friends.route.js

```diff
@@ -0,0 +1,117 @@
+const express = require('express');
+const passport = require('passport');
+const mongoose = require('mongoose');
+const router = express.Router({ mergeParams: true });
+const User = require('../../../models/user');
+
+// @route:  GET /users/:userId/friends
+// @desc:   Find and return array of friends of specified userId
+// @access: Public
+router.get('/', async (req, res) => {
+  const user = await User.findById(req.params.userId);
```

Comment:
If we use the auth middleware, we shouldn't have to get the user, as the user will be fetched from the middleware. We should also be able to only fetch the friends of the logged in user